### ISSUE TITLE
Rule suggestion - prefer toBeHidden

### ISSUE THREAD

**Issue by igorTT** [playwright-community/eslint-plugin-playwright#53]
Hi. What do you think about an optional rule 'prefer-to-be-hidden'?
https://playwright.dev/docs/1.21/test-assertions#locator-assertions-to-be-hidden

❌ Example of incorrect code for this rule:
```javascript
const locator = page.locator('.my-element');
await expect(locator).not.toBeVisible();
```

✅ Example of correct code for this rule:
```javascript
const locator = page.locator('.my-element');
await expect(locator).toBeHidden();
```

**Comment by mxschmitt** [comment]
The same for `expect(locator).not.toBeHidden();`

**Comment by mskelton** [comment]
Yeah, perhaps it could be called `no-useless-not` or something???

**Comment by igorTT** [comment]
Yes, `no-useless-not` or `avoid-useless-not`.
